benchmarks/src/main/scala/zio/chunks/ChunkArrayBenchmarks.scala

```diff
@@ -0,0 +1,48 @@
+package zio.chunks
+import zio._
+
+class ChunkArrayBenchmarks {
+
+  import java.util.concurrent.TimeUnit
+
+  import org.openjdk.jmh.annotations._
+  @State(Scope.Thread)
+  @BenchmarkMode(Array(Mode.AverageTime))
+  @OutputTimeUnit(TimeUnit.NANOSECONDS)
+  class ChunkBenchmarks {
```

Comment:
No need for this nesting, please stay consistent with other benchmarks and pull out its benchmarks to `ChunkArrayBenchmarks`.